api_tests/steps/check_pds.py

```diff
@@ -29,3 +33,43 @@ def check_asid_parameter(self, expected_status_code: int, expected_asid: list, p
         assert self.check_status_code(response, expected_status_code), \
             f"UNEXPECTED RESPONSE {response.status_code}: {response.text}"
         return True
+
+    @staticmethod
+    def check_patch_error_response(token: str, patient_id: str, expected_response: dict):
+        patient = PdsRequest(token, patient_id=patient_id, proxy="personal-demographics-pr-464")
+        patient.patch_record(op='replace', path="/gender", value="male")
+        assert patient.patched_record.get_consolidated_error() == expected_response, \
+            f"UNEXPECTED RESPONSE {patient.patched_record.status_code}: {patient.patched_record.response}"
+        return True
+
+    @staticmethod
+    def check_patch_response_code(token: str, patient_id: str, op: str, path: str, value: str, expected_status_code):
+        patient = PdsRequest(token, patient_id=patient_id, proxy="personal-demographics")
+        assert patient.patch_record(op, path, value) == expected_status_code, f"UNEXPECTED RESPONSE " \
+                                                                              f"{patient.patched_record.status_code}:" \
+                                                                              f" {patient.patched_record.response}"
+        return True
+
+    @staticmethod
+    def check_search_response(token: str, search_params: dict, expected_patient_id: Union[int, str]):
+        patient = PdsRequest(token, search_params=search_params, proxy="personal-demographics-pr-464")
```

Comment:
Can this point at just 'personal-demographics' now?

Reply:
updated